### PR TITLE
[eclipse/xtext#1631] cleanup and mark deprecations

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext#1631] cleanup and mark deprecations
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/contentassist/antlr/AntlrProposalConflictHelper.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2009 itemis AG (http://www.itemis.eu) and others.
```
Comment (cdietrich) [inline org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/contentassist/antlr/AntlrProposalConflictHelper.java]: i moved this class out of deprecated as it is still in use for new xtext projects

**Comment by cdietrich** [comment]
@szarnekow  this is just a WIP
one problem is that there is still stuff in use i dont understand.
maybe you can help me out there
(like `org.eclipse.xtext.ui.codetemplates.ui.partialEditing.PartialContentAssistContextFactory` uses of of the deprecated classes)

File: org.eclipse.xtext.ui/deprecated/org/eclipse/xtext/ui/editor/contentassist/antlr/AbstractPartialContentAssistParser.java [hunk 2]
```diff
@@ -1,3 +1,10 @@
+/*******************************************************************************
```
Comment (szarnekow) [inline org.eclipse.xtext.ui/deprecated/org/eclipse/xtext/ui/editor/contentassist/antlr/AbstractPartialContentAssistParser.java]: Appreciated attention to detail

File: org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/contentassist/antlr/AntlrProposalConflictHelper.java (hunk 1 above)
Comment (szarnekow) [reply org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/contentassist/antlr/AntlrProposalConflictHelper.java]: :+1:

**Comment by cdietrich** [comment]
@szarnekow any idea how to deal with PartialContentAssistContextFactory
i have no idea what codetemplates does

**Comment by szarnekow** [comment]
It redirects the proposal requests to the embedded languages proposal provider. We'd need to rework that to work with the xtext.ide suff to get rid of the deprecations there.

**Comment by cdietrich** [comment]
will have a look next year

**Comment by cdietrich** [comment]
@miklossy could you look into this issue?

**Comment by szarnekow** [comment]
@miklossy Do you have the time to check this out?

**Comment by miklossy** [comment]
No.

**Comment by cdietrich** [comment]
will obsolete this

**Pull request closed without merging**